docs/management/admin/host-isolation-ov

```diff
@@ -0,0 +1,69 @@
+[[host-isolation-ov]]
+[chapter, role="xpack"]
+= Host isolation
+
+Host isolation allows you to isolate Windows and macOS hosts from your network, blocking communication with any other hosts on your network until the host is released. Isolating a host is useful when responding to malicious activity or to prevent potential attacks, as it prevents lateral movement across other hosts.
+
+NOTE: To isolate and release hosts, you must have the built-in superuser role. For more information, see {ref}/built-in-users.html[Built-in users].
+
+[role="screenshot"]
+image::images/isolated-host.png[Shows a host that's been isolated]
+
+You can isolate a host via an alert attached to a case or from the Endpoints list. Once a host is successfully isolated, an *Isolated* status displays next to the *Agent Status* field, which you can view on the alert details flyout or Endpoints list table.
+
+TIP: If the request fails, verify that the Agent and your Endpoint are both online before trying again.
+
+All actions executed on a host are tracked in the host’s activity log, which you can access from the Endpoints page. See <<view-host-isolation-details, View host isolation details>> for more information.
+
+[discrete]
+[[isolate-a-host]]
+== Isolate a host
+
+To isolate a host from a case alert:
+
+. Go to *Investigate -> Cases*, then select  and click on the appropriate case to view the case activity. Ensure you are viewing a case with at least one alert attached to it.
+. Find the appropriate alert, then click the *Show alert details* button *(>)*. The alert details flyout opens.
+. Click *Take action -> Isolate*.
+. Enter a comment describing why you’re isolating the host (optional).
+. Click *Confirm*.
+
+To isolate a host from an endpoint:
+
+. Go to *Manage -> Endpoints*, then select the appropriate host name in the Hostname column. The Hostname details flyout appears.
```

Comment:
Throughout the file, some flyouts "appear" and some "open." I think they're both correct, but maybe just use a single term for consistency? My personal preference is "opens," so I edited for that throughout, but you could also change all the "opens" to "appears."
```suggestion
. Go to *Manage -> Endpoints*, then select the appropriate host name in the Hostname column. The Hostname details flyout opens.
```